hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/DispatchingEvmFrameState.java

```diff
@@ -420,4 +431,8 @@ private com.hedera.hapi.node.state.token.Account validatedAccount(final long num
         }
         return account;
     }
+
+    private UInt256 valueOrZero(@Nullable final SlotValue slotValue) {
+        return (slotValue == null) ? UInt256.ZERO : pbjToTuweniUInt256(slotValue.value());
```

Comment:
Danno mentioned that the tuweni libraries are soon to be not supported.  Nothing to do for now but something to keep in mind for later.